DRIVERS.md

```diff
@@ -0,0 +1,153 @@
+    `tests/integration/records/records_numeric_database_fixture.py`.
+    This is where you'll start to get familiar with the different
+    column types available for your database.  Be sure to be as
+    thorough as practical for your database so we can support both
+    exporting a wide variety of column types and so that we can
+    support space-efficient use on import.
+
+    For the numeric tests, when re-running you'll probably need to
+    start filling out a subclass of DBDriver.  Relevant methods:
+    `type_for_fixed_point()`, `type_for_floating_point()`,
+    `fp_constraints()`, and `integer_limits()`.
+  * `KeyError: 'mysql'` in
+    `tests/integration/records/single_db/test_records_numeric.py`:
+    There are test expectations to set here based on the numeric types
+    supported by your database.  Once you set them, you'll probably
+    need to add ad `type_for_integer()` method covering things
+    correctly.
+  * `AssertionError: ['INTEGER(11)', 'VARCHAR(3)', 'VARCHAR(3)',
+    'VARCHAR(1)', 'VARCHAR(1)', 'VARCHAR(3)', 'VARCHAR(111)', 'DATE',
+    'TIME', 'DATETIME', 'DATETIME']`: Double check the types assigned.
+    You may need to subclass DBDriver and implement to convince
+    records mover to create the types you expect.
+  * Errors from `tests/integration/records/directory_validator.py`:
+     ```console
+     AssertionError:
+     received ['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'time', 'datetime', 'datetime'],
+     expected [['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'time', 'datetime', 'datetimetz'], ['integer', 'string', 'string', 'string', 'string', 'string', 'string', 'date', 'string', 'datetime', 'datetimetz']]
+     ```
+
+     To address, make sure the types returned are as expected for this database.
+   * `KeyError: 'mysql'`:
+     `tests/integration/records/single_db/test_records_numeric.py`
+     needs to be modified to set expectations for this database type.
+     You can set this to 'bluelabs' as we haven't yet taught
+     records-mover to do bulk imports, so we have no idea what the
+     ideal records format variant is for that yet.
+   * `AssertionError` in
+     `tests/integration/records/table_validator.py`: There are various
+     checks here, including things dealing with how datetimes get
+     rendered.  Examine carefully the existing predicates defined
+     within and add new ones judiciously if it appears the behavior
+     you are seeing is correct but not currently anticipated.
+4. If there are things you see below that you know are needed from the
+   above list, but the tests are passing, consider adding an
+   integration test to match.
+5. Edit
+   `tests/integration/records/multi_db/test_records_table2table.py` to
+   include the new test database and run `./itest table2table` to run
+   tests.  Fix errors as they pop up:
+   * TBD
+7. Add support for bulk import if the database supports it TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker